fmriprep/workflows/bold.py

```diff
@@ -943,7 +947,7 @@ def init_bold_reg_wf(freesurfer, bold2t1w_dof, bold_file_size_gb,
     return workflow
 
 
-def init_bold_surf_wf(output_spaces, name='bold_surf_wf'):
+def init_bold_surf_wf(output_spaces, medial_surface_nan=False, name='bold_surf_wf'):
```

Comment:
Let's not set this by default. And you're going to want to find the instances of `init_fmriprep_wf`, `init_single_subject_wf`, `init_func_preproc_wf` and `init_bold_surf_wf` in `docs/` and add `medial_surface_nan=False` to the invocation. (Those are workflows that are created to add graphs to the docs.)

Reply:
@effigies couldn't find `init_fmriprep_wf` but I think I hit them all?